app/views/course/assessment/answer/programming/_programming.json.jbuilder

```diff
@@ -38,16 +38,19 @@ displayed_test_case_types = ['public_test']
 displayed_test_case_types << 'private_test' if show_private
 displayed_test_case_types << 'evaluation_test' if show_evaluation
 
+
+
```

Comment:
No need to introduce extra empty lines.

Reply:
@weiqingtoh The extra empty lines were introduced by mistake. I will remove the lines and update the PR.